templates/overrides/search/facets-summary-count.html.twig

```diff
@@ -4,6 +4,5 @@
  * Default theme implementation to show facets summary count.
  */
 #}
-{% if count %}
-  <span class="source-summary-count">({{ count }})</span>
-{% endif %}
+{% set count = count ?? 0 %}
```

Comment:
if the result from our live total is 0 when it should be, then there is not point in this check